### ISSUE TITLE
Drag files out of the browser into the file explorer or some other application

### ISSUE THREAD

**Issue by kruzr-app** [wailsapp/wails#3158]
### Is your feature request related to a problem? Please describe.

I am trying to create a app that converts image.webp to png etc. I thought it'd be nice if I could make it so that upon dragging the image onto something like the file explorer. But after several attempts I was unable to implement this feature

### Describe the solution you'd like

I would like a api from go or the js in which we can just set drag data or something to a file path and it would just work 

### Describe alternatives you've considered

The alternative is to not implement the feature that I've considered

**Comment by leaanthony** [comment]
This would be great. Do you know if there's support for that in the webviews? I suspect it's patchy with Windows probably having the best support for it

**Comment by astrin-app** [comment]
Yes I did, https://github.com/MicrosoftEdge/WebView2Feedback/issues/2313,
![160259076-88d4a974-0b26-46ab-8318-83fa63b61395](https://github.com/wailsapp/wails/assets/140298961/9884c555-6459-4328-8a9e-bf64e6404fe9)
But I didn't understand it I hope you could help

**Comment by leaanthony** [comment]
Hey there\!

First off, thank you for taking the time to open this issue. Community feedback like yours has been instrumental in shaping Wails into what it is today.

We're doing some housekeeping on our issue tracker as we focus on the v3 release. This issue has been open for a while, and honestly, with the project's growth and the sheer volume of feedback we receive, some things inevitably slip through the cracks. That's on us, not you.

We're closing this issue, but that doesn't mean it wasn't valuable. If this is still relevant:

- For bugs: Please test with the latest version (or v3 alpha) and open a fresh issue if it persists
- For features: Feel free to open a new issue with updated context

Thank you for being part of the Wails journey. Your patience and support mean more than you know.
